harper-core/src/linting/spelled_numbers.rs

```diff
@@ -64,14 +70,18 @@ fn spell_out_number(num: u64) -> Option<String> {
         70 => "seventy".to_string(),
         80 => "eighty".to_string(),
         90 => "ninety".to_string(),
-        100 => "one hundred".to_string(),
+        hundred if hundred % 100 == 0 => {
+            format!("{} hundred", spell_out_number(hundred / 100).unwrap())
+        }
```

Comment:
This logic should probably be expanded to include more suffixes for powers of 10 up to `10^19` so it will work for large numbers at the beginning of sentences.

(`10^19` is the largest power of 10 a u64 could represent)